Gemfile

```diff
@@ -6,3 +6,6 @@ gem 'sinatra-contrib'
 
 gem 'puma'
 gem 'json-jwt'
+
+gem "rack", ">= 2.2.8.1"
+
```

Comment:
Don't think we want to specify `rack` here as the app doesn't directly use it.

`bundle update --conservative rack` should generate a diff which just changes Gemfile.lock.

Reply:
Done, thanks.